pkg/sql/schemachanger/scdecomp/decomp.go

```diff
@@ -318,6 +322,21 @@ func (w *walkCtx) walkRelation(tbl catalog.TableDescriptor) {
 		w.backRefs.Add(fk.OriginTableID)
 		return nil
 	})
+	// Add a zone config element which is a stop gap to allow us to block
+	// operations on tables. To minimize RTT impact limit this to only
+	// tables.
+	if tbl.IsTable() {
```

Comment:
According to the element model definition that you added, zone configs can also exist for views and sequences. This line is inconsistent with that, please address.